### ISSUE TITLE
Inconsistent entry size

### ISSUE THREAD

**Issue by McFranko** [McFranko/padlock-whitepaper#3]
*"entry work" refers to the work needed to be done to submit an entry and "miner work" refers to the work that a miner must do to add their block to the blockchain.

Some spots have say the entry size is 56, and some say 48. This is because I am unsure whether the 8 byte proof of work needs to be included on chain, or whether all that needs to be done is have nodes not accept entries into the mempool with a proof of work lower than a certain threshold. 

If it's not included on chain, then miners don't have any incentive to include non spam entries. If it does have to be on chain, then it could potentially contribute to the amount of work the miner has to do (the more proof of work in the entries in a block, the less work the miner has to do to mine a block). Of course though, during high periods of congestion, then the amount of work each entry needs could spike. That would result in a bad ux, as users would need to wait long periods of time for their wallet to find a proof of work for their entry, however, it would greatly decrease the likelyhood of spam transactions being mined. It also has the negative effect if increasing entry size by 17%, therefore decreasing throughput.

It also has the potential to make it increasingly difficult for people with worse hardware to have their transactions included in a block.

One option is to try both methods on testnets, and put a very small cap on the block limit, like 10 kilobytes (in order to artificially create congestion), and see what happens to entry work requirements.

It's also very possible that work requirements would still go up drastically in high congestion periods without having entry work contribute to miner work, just because nodes would increase the amount of work required to even enter the mempool.

# TLDR
So to sum up my wall of text, including entry work results in less throughput, could make ux worse specifically for people with worse hardware, but results in much better DOS protection. Only having entry work be required to enter the mempool results in better throughput, would worsen ux to a lesser extent than having entry work being on-chain, but have worse DOS protection. Both methods could also be tested on testnets with a low block size cap in order to create artificial congestion. What are your thoughts on this?
